docs/setuptools.txt

```diff
@@ -2475,10 +2475,10 @@ Metadata
 Key                             Aliases            Type               Minumum Version
 ==============================  =================  =================  ===============
 name                                               str
-version                                            attr:, file:, str
+version                                            attr:, file:, str  39.2.0
```

Comment:
This one is tricky, because `version` has almost certainly always been supported, but the `file` attribute has only been supported since version 39.2.0.

I think maybe we should do a footnote under "Minimum Version" with an explanation at the bottom of the table, similar to the way footnotes are handled [here](https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior).